_docs-sources/foundations/landing-zone/manage-accounts.md

```diff
@@ -35,40 +35,33 @@ Below you'll find a table with common AWS account operations and the Gruntwork r
 
 ### Remove an AWS account
 
-We recommend using the AWS Console's Organizations interface for closing an account and destroying all of its resources, due to current flakiness of the AWS Service Catalog when performing deletes via OpenTofu/Terraform providers. See the steps below for the complete sequence of actions to remove an account.
+We recommend the following three step procedure to close an AWS account managed by Devops Foundations:
 
-:::note
-AWS Organizations will permanently [close an account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-closing.html) after 90 days.
-:::
+1. [Cleanup the OpenTofu/Terraform State for the Control Tower module](#1-cleanup-opentofuterraform-state-for-the-control-tower-module)
+1. [Cleanup Infrastructure Code](#2-cleanup-infrastructure-code)
+1. [Close the Account with Clickops](#3-close-the-accounts-in-aws-organizations)
 
+Note that we are recommending that ultimately accounts be closed with ClickOps instead of using Gruntwork Pipelines. Removing the account via pipelines by deleting the account request file can and often does work, however the underlying AWS Service Catalog that we use to interact with Control Tower and deprovision the account is not reliable and often returns spurious errors that can require multiple retries to complete successfully.  The procedure here is fundamentally about working around that unreliability.
```

Comment:
```suggestion
We are recommending that accounts be closed with ClickOps instead of using Gruntwork Pipelines. Removing the account via pipelines by deleting the account request file can and often does work, however the underlying AWS Service Catalog that we use to interact with Control Tower and deprovision the account is [not reliable](https://github.com/hashicorp/terraform-provider-aws/issues/31705) and often returns spurious errors that can require multiple retries to complete successfully.  The procedure here is fundamentally about working around that unreliability.
```